Sources/Core/Models/RangeFilterConfiguration.swift

```diff
@@ -3,9 +3,12 @@
 //
 
 public struct RangeFilterConfiguration: Equatable {
+    public typealias StepInterval = (range: Range<Int>, increment: Int)
+
     public enum ValueKind {
         case incremented(Int)
         case steps([Int])
+        case intervals(array: [StepInterval], defaultIncrement: Int)
```

Comment:
I think I would prefer to use an interval with an open range instead of using default increment. Like this: 
```swift
[
    (range: 0 ..< 500, increment: 50),
    (range: 500 ... , increment: 100)
]
```
Do you think there are situations where the intervals are not continues?

Reply:
I agree it looks better, but `500 ...` is  `PartialRangeFrom` type which is not convertible to `Range`. We could of course introduce a new type instead:

```swift
struct StepInterval {
   let from: Int
   let to: Int
   let increment: Int
}
```

Reply:
Right! I'm not sure which I like better :P It is nice to use the `Range` operator.. Could perform some Swift magic and make our own operator, or overload the `...` :P

Reply:
What about using an array of `(from: Int, increment: Int)` and define a range between the elements of the array? if it is always going to be continues it will be fine. I don't know if it is better though...

Reply:
Great idea! I think it's easier to read 👍